src/components/BookCard.jsx

```diff
@@ -20,26 +20,37 @@ const BookCard = () => {
   const renderBooks = books.map((book) => (
     <div key={book.item_id} className="bookContainer">
       <div className="bookTitle">
```

Comment:
- [ ] Dear @Cisco-the-wayword, Though your design is great, I can see you have no placeholder book, and I'm unable to add a new book. This prevents me from checking how your books will be rendered in the age against the original design. Please make sure two have some books on your page styled as shown in the original design. 

### Your design 
![image](https://github.com/Cisco-the-wayword/Bookstore/assets/69077061/9dafb86d-36e8-4196-8bfa-f057ef196ff0)
### Original design 
![image](https://github.com/Cisco-the-wayword/Bookstore/assets/69077061/60717cd4-4df2-4a58-ac95-36813b742c8a)